doc/source/_static/numpy.css

```diff
@@ -0,0 +1,40 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap');
+
+.navbar-brand img {
+   height: 75px;
+}
+.navbar-brand {
+   height: 75px;
+}
+
+body {
+  font-family: 'Open Sans', sans-serif;
+  color:#4A4A4A; /* numpy.org body color */
+}
+
+pre, code {
+  font-size: 100%;
+  line-height: 155%;
+}
+
+h1 {
+  font-style: "Lato", sans-serif;
+  color: #013243; /* warm black */
+  font-weight: 700;
+  letter-spacing: -.04em;
+  text-align: right;
+  margin-top: 3rem;
+  margin-bottom: 4rem;
```

Comment:
I'd argue this isn't numpy-specific styling, and would be inclined to avoid it just for the sake of minimizing differences from the upstream theme.

Reply:
It's the numpy.org headline typeface, body typeface, and color palette.  It's implemented via a standalone CSS file so there's minimal interaction with the upstream theme.

Reply:
I mean specifically these three lines, which are none of those things

Reply:
The PR makes a number of styling changes. I don't see much stability gained by eliminating three isolated lines or much risk in keeping them.

They're meant to make the page more attractive and readable.  If they succeed, the cost is minimal.

Reply:
> They're meant to make the page more attractive and readable. 

Why not suggest them upstream?

Reply:
For the same reason I'm not suggesting the others.

Reply:
That would be a good follow-up, but not a blocker for this PR.

Reply:
I'm not sure the right alignment is an improvement:

![image](https://user-images.githubusercontent.com/425260/94812482-b5ac4d80-03ee-11eb-97a5-dd42f409bff6.png)

The invisible `¶` symbol makes the right edge jagged and IMO kind of ugly.